Specs/Scene/ModelSpec.js

```diff
@@ -2824,20 +2824,25 @@ describe(
       })
         .then(function (m) {
           // Get the rendered color of the model before textures are loaded
-
           m.zoomTo();
           expect(scene).toRenderAndCall(function (rgba) {
             expect(rgba).not.toEqual([0, 0, 0, 255]);
             loadedColor = rgba;
           });
 
           model = m;
+
+          // Render at least once to initialize
+          scene.renderForSpecs();
+          return Promise.resolve();
```

Comment:
When already inside a `then` block, there is no need to return `Promise.resolve`, you don't need the return statement at all, the result will just be a Promise that resolves to udnefined.